tdrs-backend/tdpservice/backends.py

```diff
@@ -10,14 +10,17 @@ class OverriddenCredentialsS3Storage(S3Boto3Storage):
     credentials and regions between different S3 storage classes.
     """
 
+    location = settings.APP_NAME
```

Comment:
I ended up adding this cause AWS_LOCATION usually sets this variable, but something is happening I don't understand that our app wasn't acknowledging this setting when I tried to change it

Reply:
AWS_LOCATION is supose to set this on the super class, but something about us doing this inheritance on this was causing the setting to not work.